tests/drivers/uart/uart_basic_api/testcase.yaml

```diff
@@ -19,8 +19,7 @@ tests:
     filter: CONFIG_UART_CONSOLE
     harness: keyboard
   drivers.uart.cdc_acm:
-    extra_args: OVERLAY_CONFIG="overlay-usb.conf"
-                DTC_OVERLAY_FILE="usb.overlay"
+    extra_args: SHIELD="cdc_acm_console"
```

Comment:
A question here, I like the general direction, but I'd like to know how tests will work with boards that already declare usb for value of `zephyr,console` chosen.

Reply:
AFAIK last property definition wins. With this PR, there are no more boards that define chosen property with reference to "zephyr,cdc-acm-uart" compatible node. But it works even if it is the case and shield is used, then there would be two CDC ACM devices, one of them would be used by the test.